### PR TITLE
Replace buttons & components from blade-components | Small (responsive) fixes

### PR DESCRIPTION
**Pull request by Roene-JustBetter**
- Replaced buttons and inputs from the Rapidez/blade-components
- Deleted old buttons & inputs
- Small responsive frontend fixes. 
----- 
I have changed the radio and checkbox to a variant because this are not really the default checkboxes or radio buttons. 
Scherm­afbeelding 2024-12-10 om 11 56 28
Scherm­afbeelding 2024-12-10 om 11 56 33

### CODE REVIEW

**Comment by royduin** [comment]
Waiting for at least a good review from @jordythevulder or @JimmyHoenderdaal

File: resources/views/components/input/radio/variant.blade.php
Comment (Roene-JustBetter) [reply resources/views/components/input/radio/variant.blade.php]: Yes, I think this is a good point. I have changed it to tile.

**Review by jordythevulder — APPROVED** [review]

**Pull request merged**
